website_and_docs/content/blog/2022/python_locators_se4.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Locate Your Locators - Python Bindings Changes Upcoming"
+linkTitle: "Locate Your Locators Python Bindings Changes Upcoming"
+date: 2022-02-01
+tags: ["webdriver", "python"]
+categories: ["general"]
+author: Josh Grant ([@joshin4colours](https://twitter.com/joshin4colours))
+description: >
+  Updating Your Locators for Changes in the Pythom Bindings in Selenium 4
+---
+
+(This article was originally posted [here](https://simplythetest.tumblr.com/post/674917293614039040/locate-your-locators-python-bindings-changes). Thanks to the Selenium core contributors for adding this here!)
+
+In real estate, the mantra for finding a new house or office space is "location, location, location!". It could be said that when working with Selenium, a critical aspect of writing tests is "locators, locators, locators!". Having a robust locator strategy - in your app under test and in your test framework - is highly important for effective testing.
+
+If you are a Pythonista like myself and using Selenium for your test automation, then there are some important changes coming to how locators are defined and used.
```

Comment:
Could you please break the lines to about 100 chars per line?
https://www.selenium.dev/documentation/about/style/#line-length